packages/gatsby-source-shopify/README.md

```diff
@@ -1,813 +1,282 @@
-# 📣 Looking for the latest version of `gatsby-source-shopify`?
+<p align="center">
+  <a href="https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-shopify/LICENSE">
+    <img src="https://img.shields.io/badge/license-MIT-blue.svg" alt="Gatsby and gatsby-source-shopify are released under the MIT license." />
+  </a>
+  <a href="https://www.npmjs.com/package/gatsby-source-shopify">
+    <img src="https://img.shields.io/npm/v/gatsby-source-shopify.svg" alt="Current npm package version." />
+  </a>
+  <a href="https://npmcharts.com/compare/gatsby-source-shopify?minimal=true">
+    <img src="https://img.shields.io/npm/dm/gatsby-source-shopify.svg" alt="Downloads per month on npm." />
+  </a>
+  <a href="https://npmcharts.com/compare/gatsby-source-shopify?minimal=true">
+    <img src="https://img.shields.io/npm/dt/gatsby-source-shopify.svg" alt="Total downloads on npm." />
+  </a>
+  <a href="https://gatsbyjs.com/contributing/how-to-contribute/">
+    <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" alt="PRs welcome!" />
+  </a>
+  <a href="https://twitter.com/intent/follow?screen_name=gatsbyjs">
+    <img src="https://img.shields.io/twitter/follow/gatsbyjs.svg?label=Follow%20@gatsbyjs" alt="Follow @gatsbyjs" />
+  </a>
+</p>
 
-👉 Get started at [`gatsbyjs/gatsby-source-shopify`](https://github.com/gatsbyjs/gatsby-source-shopify)
+# gatsby-source-shopify
 
-- Support for incremental builds
-- Scale up to 10k products
-- Up to 16x faster builds
+A scalable solution for sourcing data from Shopify.
 
-`gatsbyjs/gatsby-source-shopify` will replace the below package with the next minor release of `gatsbyjs/gatsby`.
+This plugin works by leveraging [Shopify's bulk operations API][bulk-operations], which allows us to process large amounts of data at once. This gives us a more resilient and reliable build process. It also enables incremental builds so that your site can build quickly when you change your data in Shopify.
```

Comment:
```suggestion
This plugin works by leveraging [Shopify's bulk operations API][bulk-operations], which allows it to process large amounts of data at once. This gives it a more resilient and reliable build process. It also enables incremental builds so that your site can build quickly when you change your data in Shopify.
```